bravado/mapping/http_future.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (c) 2013, Digium, Inc.
+# Copyright (c) 2014, Yelp, Inc.
+#
+
+DEFAULT_TIMEOUT_S = 5.0
+
+
+class HttpFuture(object):
```

Comment:
This can just be in http_client.py unless you've got a good reason to make it a separate module.